admin/spam/views.py

```diff
@@ -1,47 +1,149 @@
 from django.shortcuts import render
-from django.core.paginator import Paginator, EmptyPage, PageNotAnInteger
 from django.contrib.auth.decorators import login_required
+from django.views.generic import FormView, ListView
+from django.utils.decorators import method_decorator
+from django.core.urlresolvers import reverse
+from django.views.defaults import page_not_found
 
 from modularodm import Q
 from website.project.model import Comment
 
 from .serializers import serialize_comment
+from .forms import ConfirmForm
 
 
-def get_spam_list():
-    query = (
-        Q('reports', 'ne', {}) &
-        Q('reports', 'ne', None)
-    )
-    return Comment.find(query).sort('date_created')
+def key_order_comments(comment):
```

Comment:
Put a breakpoint in this function and it wasn't hit. Please remove.

Also, confirmed that it was sorting by most recent report first.